packages/ui-components/src/Avatar/Avatar.test.tsx

```diff
@@ -1,75 +1,91 @@
 import React from "react";
-import { shallow } from "enzyme";
+import { render, fireEvent, screen } from "@testing-library/react";
 
 import Avatar, { AvatarIntent, AvatarSize } from "./Avatar";
```

Comment:
I don't want to pollute the component with a data attribute, but I agree that having one on the component is convenient for testing. So maybe we could wrap the component from the lib to add a data attribute in this file for testing purposes. I'm imagining something like,

```jsx
import UiAvatar, { AvatarIntent, AvatarSize, AvatarProps } from "./Avatar";

const Avatar: React.FC<AvatarProps> = ({ children, ...props }) => (
  <UiAvatar data-testid="avatar" {...props}>
    {children}
  </UiAvatar>
);
```

Reply:
updated the tests @nathanstilwell